In [3]:
bold = '\033[1m' 
end = '\033[0m'

import pandas as pd
import numpy as np
from matplotlib import pyplot as plt
from IPython.display import Image, HTML, display
import math

# `EXERCISES`

# Exercise 3.1

When two dice ––– one black  with white dots, the other black with white dots ––– are tossed once, simultaneously, and the number of dots shown on each die's top face after coming to rest are recorded as an ordered pair $(n_B, n_W)$, where $n_B$ is the number on the black die, and $n_W$ the number on the white die,

**(i)** identify the **experiment**, what constitutes a **trial**, the **outcomes**, and the **sample space**.

<hr>
### `Solution:`

**Experiment:** Tossing two dice simultaneously and recording their face value  as an ordered pair $(n_B, n_W)$.

**Trial:** Performing the experiment once.

**Outcomes:** The recorded face value ordered pair.

**Sample space:** $ (1,1), (1,2), (1,3), ... (1,6); (2,1), (2,2), ... (2,6); (3,1), ... (3,6); ... (6,6) $

<hr>

**(ii)** If the sum of the numbers on the two dice is $S$, i.e.,
$$
S = n_B + n_W,
$$

enumerate all the simple events associated with the observation $S = 7$.

### `Solution:`

Possible outcomes for $S = 7$ are : $(1,6), (2,5), (3,4), (4,3), (5,2), (6,1) $

# Exercise 3.2

In an opinion poll, 20 individuals selected at random from a group of college students are asked to indicate which of three options –––*approve, disapprove, indifferent*–– best matches their individual opinions of a new campus policy.  Let $n_0$ be the number of indifferent students, $n_1$ the number that approve, and $n_2$ the number that disapprove, so that the outcome of one such opinion sample is the ordered triplet $(n_0, n_1, n_2)$.  Write mathematical expressions in terms of the numbers $n_0, n_1$, and $n_2$ for the following events:

**(i)** $A=$ \{Unanimous support for the policy\}; and $A^*$, the complement of $A$.

### `Solution:`


For unanimous support of policy, $A = \{n_1 > n_0 + n_2\}$

$A^* = \{n_1 <= n_0 + n_2\}$

**(ii)** $B=$ \{More students disapprove than approve\}; and $B^*$.

### `Solution:`

$B = \{n_2 > n_1\}$

$B^* = \{n_2 <= n_1\}$

**(iii)** $C=$ \{More students are indifferent than approve\};

### `Solution:`

$C = \{n_0 > n_1\}$

**(iv)** $D=$ \{The majority of students are indifferent \}.

### `Solution:`

$D = \{n_0 > n_1+n_2\}$

# Exercise 3.3

Given the following two sets $A$ and $B$:
$$
A = \{x: x= 1, 3, 5, 7, \ldots\} \\
B = \{x: x= 0, 2, 4, 6, \ldots\}
$$
find $A \cup B$ and $A \cap B$. 

### `Solution:`

$ A \cup B = \{x : x = 0,1,2,3,4,5, ...\} $

$ A \cap B = \phi$


In [13]:
## Programmatical results:

a = set(range(1, 100, 2))
b = set(range(0, 100, 2))

print("A ∪ B = ", a.union(b))
print("\n\nA ∩ B = ", a.intersection(b))

A ∪ B =  {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99}


A ∩ B =  set()


# Exercise 3.4

Let $A_k = \{x: 1/(k+1)\le x \le 1\}$ for $k = 1, 2, 3, \ldots$.  Find the
set $B$ defined by:
$$
B = A_1 \cup A_2 \cup A_3 \cup \ldots = \bigcup_{i=1}^{\infty} A_i
$$

### `Solution:`

With increse in $k$, $A_k$ also increses in size. Thus the set $B$, union of all sets $A_k$ for $k = 1,2,3,...$, should be equal to the largest set i.e. $A_\infty$

$\implies B = \{x : 1/(\infty + 1) \le x \le 1\}$

$\implies B = \{x : 0 \le x \le 1\}$

# Exercise 3.5

For sets $A, B, C$, subsets of the universal set $\Omega$, establish the following identities:

$$
(A \cup B)^* = A^* \cap B^* \\
(A \cap B)^* = A^* \cup B^* \\
A \cap (B \cup C) = (A \cap B) \cup (A \cap C) \\
A \cup (B \cap C) = (A \cup B) \cap (A \cup C)
$$

In [46]:
# Lets assume the sets A, B, C and universal set S

s = set(range(30))
a = set(range(0, 30, 2))
b = set(range(1, 30, 3))
c = set(range(2, 30, 3))

print("Set S =", s)
print("\nSet A =", a)
print("Set B =", b)
print("Set C =", c)


# Complement sets
a_ = s - a
b_ = s - b
c_ = s - c

print("\nSet A* =", a_)
print("Set B* =", b_)



Set S = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29}

Set A = {0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28}
Set B = {1, 4, 7, 10, 13, 16, 19, 22, 25, 28}
Set C = {2, 5, 8, 11, 14, 17, 20, 23, 26, 29}

Set A* = {1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29}
Set B* = {0, 2, 3, 5, 6, 8, 9, 11, 12, 14, 15, 17, 18, 20, 21, 23, 24, 26, 27, 29}


In [47]:
# Establishing Identity 1

ab = a.union(b)  # a union b
ab_ = s - ab  # (a union b)*

a_b_i = a_.intersection(b_)  # (a* intersection b*)

print(ab_)
print(a_b_i)

print("Is (A union B equal)* to A* intersection B* : ", ab_ == a_b_i)


# Establishing Identity 2

ab_i = a.intersection(b)  # a intersection b
ab_i_ = s - ab_i  # (a intersection b)*

a_b_ = a_.union(b_)  # (a* intersection b*)

print("\n")
print(ab_i_)
print(a_b_)

print("Is (A intersection B)* equal to A* union B* : ", ab_ == a_b_i)


# Establishing Identity 3
bc = b.union(c)
a_bc_i = a.intersection(bc)

ab_i = a.intersection(b)
ac_i = a.intersection(c)

ab_i_ac_i = ab_i.union(ac_i)

print("\n")
print(a_bc_i)
print(ab_i_ac_i)
print("Is A intersection (B union C) = (A intersection B) union (A intersection C) : ", a_bc_i == ab_i_ac_i)


# Establishing Identity 4
bc_i = b.intersection(c)
a_bc_i = a.union(bc_i)

ab = a.union(b)
ac = a.union(c)

ab_ac_i = ab.intersection(ac)

print("\n")
print(a_bc_i)
print(ab_i_ac_i)
print("Is A union (B intersection C) = (A union B) intersection (A union C) : ", a_bc_i == ab_ac_i)

{3, 5, 9, 11, 15, 17, 21, 23, 27, 29}
{3, 5, 9, 11, 15, 17, 21, 23, 27, 29}
Is (A union B equal)* to A* intersection B* :  True


{0, 1, 2, 3, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 17, 18, 19, 20, 21, 23, 24, 25, 26, 27, 29}
{0, 1, 2, 3, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 17, 18, 19, 20, 21, 23, 24, 25, 26, 27, 29}
Is (A intersection B)* equal to A* union B* :  True


{2, 4, 8, 10, 14, 16, 20, 22, 26, 28}
{2, 4, 8, 10, 14, 16, 20, 22, 26, 28}
Is A intersection (B union C) = (A intersection B) union (A intersection C) :  True


{0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28}
{2, 4, 8, 10, 14, 16, 20, 22, 26, 28}
Is A union (B intersection C) = (A union B) intersection (A union C) :  True


# Exercise 3.6

For every pair of sets $A, B$, subsets of the *Sample Space* $\Omega$
upon which the probability set function $P(.)$ has been defined, prove that:

$$
P(A \cup B) = P(A) + P(B) - P(A \cap B)
$$

In [101]:
# Let all the outcomes be equiprobable

s = set(range(10))
p_of_each_outcome = 1/len(s)

a = set(range(0, 10, 2))
b = set(range(0, 10, 3))

k = list(a)
k.extend(list(b))

print("A =", a)
print("B =", b)
print("A union B =", a.union(b))
print("A + B = {", *sorted(k), "}")
print("A intersection B =", a.intersection(b))

print("\nAs we can see, (A union B) =  A + B - (A intersection B)")

pa = p_of_each_outcome*len(a)
pb = p_of_each_outcome*len(b)

print("\n")
print("P(A) =", pa)
print("P(B) =", pb)
print("P(A) + P(B) =", pa+pb)
print("P(A intersection B) =", p_of_each_outcome*len(a.intersection(b)))

print("\nP(A union B) =", 
      np.round(p_of_each_outcome*len(a.union(b)), 2),
     "= P(A) + P(B) - P(A intersection B) =", 
      pa + pb - p_of_each_outcome*len(a.intersection(b))
     )



A = {0, 2, 4, 6, 8}
B = {0, 9, 3, 6}
A union B = {0, 2, 3, 4, 6, 8, 9}
A + B = { 0 0 2 3 4 6 6 8 9 }
A intersection B = {0, 6}

As we can see, (A union B) =  A + B - (A intersection B)


P(A) = 0.5
P(B) = 0.4
P(A) + P(B) = 0.9
P(A intersection B) = 0.2

P(A union B) = 0.7 = P(A) + P(B) - P(A intersection B) = 0.7


# Exercise 3.7

In a certain engineering research and development company, apart from the
support staff which number 25, all other employees are either engineers or
statisticians or both.  The total number of employees (including the support
staff) is 100.  Of these, 50 are engineers, and 40 are statisticians; the number
of employees that are both engineers and statisticians is not given.  Find the probability that  an employee chosen at random is **not** one of those classified as being both an engineer and a
statistician.

In [104]:
total = 100
support = 25
other = total - support

engineers = 50
statisticians = 40

both = engineers + statisticians - other

print("P(not both statisticians and engineers) =", 1 - (both/total))

P(not both statisticians and engineers) = 0.85


# Exercise 3.8

For every set $A$, let the set function $Q(.)$ be defined as follows:

$$
Q(A) = \sum_{A} f(x)
$$
where
$$
f(x) = \left(\frac{2}{3}\right) \left( \frac{1}{3}\right)^x ; x = 0,1,2,\ldots
$$

If $A_1 = \{x: x= 0,1,2,3\}$ and $A_2 = \{x: x= 0,1,2,3, \ldots\}$ find $Q(A_1)$
and $Q(A_2)$.

In [126]:
def fx(x):
    return ((2/3)*((1/3)**x))

def QA(A):
    result = 0
    for x in A:
        result += fx(x)
        
    return result

A1 = [0,1,2,3]
A2 = list(range(100))

print("Q(A1) =", QA(A1))
print("Q(A2) =", QA(A2))
print("Note: Q(A2) tends to 1 when x tends to infinity.")

Q(A1) = 0.9876543209876543
Q(A2) = 0.9999999999999997
Note: Q(A2) tends to 1 when x tends to infinity.


# Exercise 3.9

Let the sample space for a certain experiment be $\Omega=\{\omega: 0 <
\omega < \infty\}$.  Let $A \subset \Omega$ represent the event $A = \{ \omega:
4 < \omega < \infty\}$.  If the probability set function $P(A)$ is defined for
any subset of the sample space according to:

$$
P(A) = \int_A e^{-x} dx
$$
evaluate $P(A), P(A^*), P(A \cup A^*)$

<hr>
### `Solution:`

Given $A = \{\omega : 4 < \omega < \infty\}$

and $P(A) = \int_A e^{-x} dx \\
\implies P(A) = \int_4^\infty e^{-x} dx = \left.\left(-e^{-x}\right)\right|_4^\infty \\
\implies P(A) = -e^{-\infty} - \left(-e^{-4}\right) = 0 + e^{-4} \\
\implies P(A) = e^{-4}
$

<br><br>

$A^* = \Omega - A \\
\implies A^* = \{\omega : 0 < \omega < \infty\} - \{\omega : 4 < \omega < \infty\} = \{\omega : 0 < \omega < 4\} \\
\implies P(A^*) = \int_0^4 e^{-x} dx \\
\implies P(A^*) = -e^{-4} - \left(-e^{-0}\right) \\
\implies P(A^*) = 1 - e^{-4}
$

**NOTE:** By definition, for any event $A$, $P(A) + P(A^*) = 1 \implies P(A^*) = 1 - P(A)$. The above calculations just proved the same result.

<br><br>

$
A \cup A* = \{\omega : 0 < \omega < 4\} + \{\omega : 4 < \omega < \infty\} \\
\implies A \cup A^* = \{\omega : 0 < \omega < \infty\} \\
\implies P(A \cup A^*) = \int_0^\infty e^{-x} dx = -e^{-\infty} - \left(-e^{-0}\right) \\
\implies P(A \cup A^*) = 1
$

**NOTE:** By definition, for any event $A$, $P(A \cup A^*) = 1$. The above calculations proved the same result.

<hr>

# Exercise 3.10

For the experiment of rolling two dice---one black  with white dots, the other black with white dots---once, simultaneously, presented in Exercise 3.1, first obtain $\Omega$, the sample space, and, by assigning equal probability to each of the outcomes.

<hr>
### `Solution:`
**Sample Space $\Omega = \{(1,1),(1,2), ... (1,6),(2,1), ... (6,6)\}$**

$n = |\Omega| = 36$

Probability for each outcome, $p = \frac{1}{36}$

<hr>

determine the probability of the following events:

(i) $A= \{n_B + n_W = 7 \}$, i.e. the sum is 7;

<hr>
### `Solution:`

$A = \{(1,6), (2,5), (3,4), (4,3), (5,2), (6,1)\}$

$P(A) = p*|A|$

$\implies P(A) = \frac{1}{36} * 6  = \frac{1}{6}$

<hr>

(ii) $B= \{n_B < n_W \}$;

<hr>
### `Solution:`

$B = \{(1,2), (1,3), (1,4), (1,5), (1,6), (2,3), (2,4), (2,5), (2,6), (3,4), (3,5), (3,6), (4,5), (4,6), (5,6)\}$

$P(B) = p*|B|$

$\implies P(B) = \frac{1}{36} * 15  = \frac{5}{12}$

<hr>

(iii) $B^*$, the complement of $B$;

<hr>
### `Solution:`

$P(B^*) = 1 - P(B)$

$\implies P(B^*) = 1 - \frac{5}{12} = \frac{7}{12}$

<hr>

(iv) $C = \{n_B = n_W \}$, i.e. the two dice show the same number;

<hr>
### `Solution:`

$C = \{(1,1), (2,2), (3,3), (4,4), (5,5), (6,6)\}$

$P(C) = p*|C|$

$\implies P(C) = \frac{1}{36} * 6  = \frac{1}{6}$

<hr>

(v) $D = \{n_B + n_W = 5 \mbox{ or } 9\}$.

<hr>
### `Solution:`

$D = \{(1,4), (2,3), (3,2), (4,1), (3,6), (4,5), (5,4), (6,3)\}$

$P(D) = p*|D|$

$\implies P(D) = \frac{1}{36} * 8  = \frac{2}{9}$

<hr>

# Exercise 3.11

A black velvet bag contains three red balls and three green balls.  Each experiment involves drawing two balls at once, simultaneously, and recording their colors, $R$ for red, and $G$ for green.

(i) Obtain the sample space, assuming that balls of the same color are indistinguishable.

<hr>
### `Solution:`

$ Sample Space = \{(R, G), (G, R), (R, R), (G, G)\}$

<hr>

(ii) Upon assigning equal probability to each element in the sample space,  determine the probability of drawing two balls of different colors.

<hr>
### `Solution:`

Different colour outcomes, D = $\{(R, G), (G, R)\}$

$P(D) = \frac{1}{4} * |D| $

$\implies P(D) = \frac{1}{4} * 2 = \frac{1}{2}$

<hr>

(iii) If the balls are distinguishable and numbered from 1 to 6, and if the two balls are drawn sequentially, not simultaneously, now obtain the sample space and from this determine the probability of drawing two balls of different colors.

<hr>
### `Solution:`

**New Sample Space, N** = $\{(1,2), (1,3), ... (1,6), (2,1), (2,3), ... (2,6), ... (6,5)\}$

$|N|$ = 30

Each ball can be paired with 3 other balls of different color. And we have 6 balls. So, number of possible outcomes for the given event $ = 6 * 3 = 18 $

$\implies P$(Balls with different colors) = $\frac{18}{30} = \frac{3}{5}$

<hr>

# Exercise 3.12

An experiment is performed by selecting a card from an ordinary deck of 52
playing cards.  The outcome, $\omega$, is the type of card chosen, classified
as: `Ace,` `King,` `Queen,` `Jack,` and `others.`  The
random variable $X(\omega)$ assigns the number 4 to the outcome if $\omega$ is
an `Ace;` $X(\omega) = 3$ if the outcome is a `King`; $X(\omega)=2$ if the
outcome is a `Queen,` and $X(\omega)=1$ if the outcome is a `Jack`;
$X(\omega)=0$ for all other outcomes.

(i) What is the space $V$ of this random variable?

<hr>
### `Solution:`

$$
V = \left\{
\begin{array}{ll}
4; & Ace \\
3; & King \\
2; & Queen \\
1; & Jack \\
0; & \mbox{Otherwise} \\
\end{array}
\right.
$$
<hr>

(ii) If the probability set function $P(\Gamma)$ defined on the subsets of the
original sample space $\Omega$ assigns a probability $1/52$ to each of these
outcomes, describe the induced probability set function $P_X(A)$ induced on all
the subsets of the space $V$ by this random variable.

<hr>
### `Solution:`

$$
P_X(A) = \left\{ 
\begin{array}{ll}
\frac{1}{52}*4 = \frac{1}{13}; & X=4 \\
\frac{1}{52}*4 = \frac{1}{13}; & X=3 \\
\frac{1}{52}*4 = \frac{1}{13}; & X=2 \\
\frac{1}{52}*4 = \frac{1}{13}; & X=1 \\
\frac{1}{52}*36 = \frac{9}{13}; & X=0
\end{array} 
\right.
$$
<hr>

(iii) Describe a physical (scientific or engineering) problem for which the
above would be a good surrogate `model.`

# Exercise 3.13

Obtain the sample space, $\Omega$, for the experiment involving tossing a fair coin 4 times. Upon assigning equal probability to each outcome, determine the probabilities of obtaining, 0, 1, 2, 3, or 4 heads.  Confirm that your result is consistent with the postulate that the probability model for this phenomenon is given by the probability distribution function:
    $$
    f(x) = \frac{n!}{x!(n-x)!}p^x (1-p)^{n-x}
    $$
    
where $f(x)$ is the probability of obtaining $x$ heads in $n=4$ tosses, and $p= {1\over2}$ is the probability of obtaining a head in a *single* toss of the coin. (See Chapter 8.)

<hr>
### `Solution:`

**Sample Space, $\Omega$** = $
\left\{ \\
(H,H,H,H), \\
(H,H,H,T), \\
(H,H,T,H), \\
(H,H,T,T), \\
(H,T,H,H), \\
(H,T,H,T), \\
(H,T,T,H), \\
(H,T,T,T), \\
(T,H,H,H), \\
(T,H,H,T), \\
(T,H,T,H), \\
(T,H,T,T), \\
(T,T,H,H), \\
(T,T,H,T), \\
(T,T,T,H), \\
(T,T,T,T) \\
\right\}$

$|\Omega|$ = 16

Assigning equal probability to each outcome $\implies P$(each outcome)$ = \frac{1}{16}$

<br>
Now,

Probability of obtaining x heads (for x = 0,1,2,3,4) denoted by $P(x)$ is:

$
P(0) = P(\{(T,T,T,T)\}) \\
\implies P(0) = \frac{1}{16}
$

$
P(1) = P(\{(H,T,T,T), (T,H,T,T), (T,T,H,T), (T,T,T,H)\}) \\
\implies P(1) = \frac{4}{16}
$


$
P(2) = P(\{(H,H,T,T), (T,H,H,T), (T,T,H,H), (H,T,T,H), (H,T,H,T), (T,H,T,H)\}) \\
\implies P(2) = \frac{6}{16}
$


$
P(3) = P(\{T,H,H,H), (H,T,H,H), (H,H,T,H), (H,H,H,T)\}) \\
\implies P(3) = \frac{4}{16}
$


$
P(4) = P(\{(H,H,H,H)\}) \\
\implies P(4) = \frac{1}{16}
$

<br><br>

According to the given probability distribution function:
$$
f(x) = \frac{4!}{x!(4-x)!}\frac{1}{2}^x (1-\frac{1}{2})^{4-x}
$$

The probabilities are:

$
P(0) = \frac{4!}{0!(4-0)!}\frac{1}{2}^0 (1-\frac{1}{2})^{4-0} = \left(\frac{1}{2}\right)^4 \\
\implies P(0) = \frac{1}{16}
$


$
P(1) = \frac{4!}{1!(4-1)!}\frac{1}{2}^1 (1-\frac{1}{2})^{4-1} = 4*\frac{1}{2} \left(\frac{1}{2}\right)^{3} \\
\implies P(1) = \frac{4}{16}
$


$
P(2) = \frac{4!}{2!(4-2)!}\frac{1}{2}^2 (1-\frac{1}{2})^{4-2} = 6*\frac{1}{2}^2 \left(\frac{1}{2}\right)^{2} \\
\implies P(2) = \frac{6}{16}
$


$
P(3) = \frac{4!}{3!(4-3)!}\frac{1}{2}^3 (1-\frac{1}{2})^{4-3} = 4*\frac{1}{2}^3 \left(\frac{1}{2}\right)^{1} \\
\implies P(3) = \frac{4}{16}
$


$
P(4) = \frac{4!}{4!(4-4)!}\frac{1}{2}^4 (1-\frac{1}{2})^{4-4} = \frac{1}{2}^4 \left(\frac{1}{2}\right)^{0} \\
\implies P(4) = \frac{1}{16}
$

<br>
As we can see, both methods yeild the same results.

<hr>

# Exercise 3.14

In the fall of 2007,  $k$ students born in 1989 attended an all-freshman introductory general engineering class at the University of Delaware.  Confirm that if $p$ is the probability that **at least two** of the students have the same birthday then: 
<br> <br>
    $$
    1-p = \frac{365!}{(365-k)!}{1 \over (365)^k}
    $$
    
Show that for a class with 23 or more students born in 1989, the probability of at least 2 students sharing the same birthday, is more than 1/2, i.e., if $k>23$ then $p>1/2$.

# Exercise 3.15

Six simple events, with probabilities $P(E_1)= 0.11; P(E_2) = P(E_5) = 0.20; P(E_3) = 0.25; P(E_4) = 0.09; P(E_6) = 0.15$, constitute the entire set of outcomes of an experiment.  The following events are of interest: <br><br>
$$
A = \{E_1, E_2\}; \; B = \{E_2, E_3, E_4\}; \; C = \{E_5, E_6\}; \; D= \{ E_1, E_2, E_5 \}
$$

Determine the following probabilities:

(i) $P(A), P(B), P(C), P(D)$;


<hr>
### `Solution:`

$
P(A) = P(E_1) + P(E_2) = 0.11 + 0.20 \\
\implies P(A) = 0.31
$

<br>
$
P(B) = P(E_2) + P(E_3) + P(E_4) = 0.20 + 0.25 + 0.09 \\
\implies P(B) = 0.54
$

<br>
$
P(C) = P(E_5) + P(E_6) = 0.20 + 0.15 \\
\implies P(C) = 0.35
$

<br>
$
P(D) = P(E_1) + P(E_2) + P(E_5) = 0.11 + 0.20 + 0.20 \\
\implies P(D) = 0.51
$



<hr>

(ii) $P(A \cup B), P(A\cap B); P(A \cup D), P(A \cap D); P(B\cup C), P(B\cap C)$;

<hr>
### `Solution:`

$
P(A \cup B) = P(\{E_1, E_2, E_3, E_4\}) = 0.11 + 0.20 + 0.25 + 0.09 \\
\implies P(A \cup B) = 0.65
$

<br>
$
P(A \cap B) = P(\{E_2\}) = 0.20 \\
\implies P(A \cap B) = 0.20
$

<br>
$
P(A \cup D) = P(\{E_1, E_2, E_5\}) = 0.11 + 0.20 + 0.20 \\
\implies P(A \cup D) = 0.51
$

<br>
$
P(A \cap D) = P(\{E_1, E_2\}) = 0.11 + 0.20 \\
\implies P(A \cap D) = 0.31
$

<br>
$
P(B \cup C) = P(\{E_2, E_3, E_4, E_5, E_6\}) = 0.20 + 0.25 + 0.09 + 0.15 + 0.20 \\
\implies P(B \cup C) = 0.89
$

<br>
$
P(B \cap C) = P(\{\phi\}) = 0 \\
\implies P(B \cap C) = 0
$


<hr>

(iii) $P(B|A), P(A|B)$; $P(B|C), P(D|C)$

<hr>
### `Solution:`

$
P(B|A) = \frac{P(B \cap A)}{P(A)} = \frac{0.20}{0.31} \\
\implies P(B|A) = 0.64516
$

<br>
$
P(B|A) = \frac{P(B \cap A)}{P(B)} = \frac{0.20}{0.54} \\
\implies P(B|A) = 0.37037
$

<br>
$
P(B|C) = \frac{P(B \cap C)}{P(C)} = \frac{0}{0.35} \\
\implies P(B|C) = 0
$

<br>
$
P(C|D) = \frac{P(C \cap D)}{P(D)} = \frac{0.20}{0.51} \\
\implies P(C|D) = 0.39215
$


<hr>

Which of the events $A, B, C$ and $D$ are mutually exclusive?

<hr>
### `Solution:`

Since $A \cap C = \phi$ and $B \cap C = \phi$, the events A and C and the events B and C are mutually exclusive. 

<hr>

# Exercise 3.16

Assuming that giving birth to a boy or a girl is equally likely, and further, that no multiple births have occurred, first, determine the probability of a family having three boys in a row. Now consider the conjecture (based on empirical data) that, for a family that has already had two boys in a row, the probability of having a third boy is 0.8. Under these conditions, what is now the probability of a family having three boys in a row?

# Exercise 3.17

As a follow-up to the concept of independence of two events $A$ and $B$,

1. Event $A$ is said to be `attracted` to  event $B$ if

        P(A|B) > P(A)

        
2. Event $A$ is said to be `repelled` by event $B$ if

        P(A|B) < P(A)
        
(Of course, when $P(A|B) = P(A)$, the two events have been previously identified as independent.)  Establish the result that if $B$ attracts $A$, then: 

(i) $A$ attracts $B$ (mutual attraction)

<hr>
### `Solution:`

Given : B attracts A 

i.e.,
$
P(A) > P(A\,|\,B) \\
\implies P(A) > \frac{P(A \,\cap\, B)}{P(B)} \\
\implies P(B) > \frac{P(A \,\cap\, B)}{P(A)} \\
\implies P(B) > P(B\,|\,A) \\
\implies A \, attracts \, B \\
\implies Mutual \, Attraction
$


<hr>

(ii) $B^*$ repels $A$.

# Exercise 3.18

Show that if $A$ and $B$ are independent, then $A^*$ and $B^*$ are also independent.

Given:
$P(A|B) = P(A) \,\,\,\,\,\,\,$ (Property of Independent Events)

$
\implies \frac{P(A \cap B)}{P(B)} = P(A) \\
\implies P(A \cap B) = P(A)*P(B) \,\,\,\,\,\,\, - Eq (1) \\
$

We also know that:

$
P(A^*) = 1 - P(A) \\
P(B^*) = 1 - P(B) 
$

So,

$
P(A^*)P(B^*) = [1 - P(A)][1 - P(B)] \\
\implies P(A^*)P(B^*) = 1 - P(A) - P(B) + P(A)P(B) \\
\implies P(A^*)P(B^*) = 1 - P(A) - P(B) + P(A \cap B) \,\,\,\,\,\,\, - (using\,Eq (1)) \\
\implies P(A^*)P(B^*) = 1 - [P(A) + P(B) - P(A \cap B)] \\
\implies P(A^*)P(B^*) = 1 - P(A \cup B) \\
\implies P(A^*)P(B^*) = P(A \cup B)^* \\
\implies P(A^*)P(B^*) = P(A^* \cap B^*) \,\,\,\,\,\,\, - (using\,De\,Morgan's\,Law) \\
$

$\implies A^*$ and $B^*$ are independent events.





# Exercise 3.19

Show that for two events $A$ and $B$, $P(A \cup B|A \cap B) \le P(A\cap B|A)$. State the condition for equality.

# Exercise 3.20

An exchange student from Switzerland, who is male, has been assigned to be your partner in an introductory psychology class. As part of a class assignment, he responds to your question about his family by stating only that he comes from a family of two children, without specifying whether he is the older or the younger.  What is the probability that his sibling is female? Assume equal probability of having a boy or girl.  Why does this result seem counterintuitive at first?

# Exercise 3.21

A system consisting of two components $A$ and $B$ that are connected in **series** functions if **both** of them  function.  If $P(A)$, the probability that component $A$ functions, is 0.99, and the  probability that component $B$ functions is 0.90, find the probability that this series system functions, assuming that whether one component functions or not is independent of the status of the other component.  If these components are connected in **parallel**, the system fails (i.e., will **not** function) only if **both** components fail.  Assuming independence, determine the probability that the parallel system functions.  Which probability is higher and why is it reasonable to expect a higher probability from the system in question?

# Exercise 3.22

The functioning status of a complex system that consists of several components arranged in series and parallel and with cross-links (i.e., whether the system functions or not) can be determined from the status of a `keystone` component,  $C_k$.  If the probability that the keystone component for a particular system functions is given as $P(C_k) = 0.9$ and the probability that the the system function when the keystone functions, $P(S|C_k)$, is given as 0.9, with the complementary probability that the system functions when the keystone **does not** function, $P(S|C^*_k)$, given as 0.8, find the unconditional probability, $P(S)$, that the system functions.